### PR TITLE
Adding GrandpaJustification custom type

### PR DESCRIPTION
**Pull request by hbulgarini**
Closes https://github.com/paritytech/parity-bridges-common/issues/872

### CODE REVIEW

File: deployments/types/common.json [hunk 1]
```diff
@@ -1,138 +1,159 @@
...
-		"unrewarded_relayer_entries": "MessageNonce",
-		"messages_in_oldest_entry": "MessageNonce",
-		"total_messages": "MessageNonce"
-	},
-	"AncestryProof": "()",
-	"MessageFeeData": {
-		"lane_id": "LaneId",
-		"payload": "OutboundPayload"
-	}
+  "--1": "Millau Types",
```
Comment (tomusdrw) [inline deployments/types/common.json:2]: Please re-format the file back according to `.editorconfig` or change the formatting there as well.

**Review by tomusdrw — CHANGES_REQUESTED** [review]
Unnecessary re-format and also you need to re-generate concrete type files with `deployments/types/build.sh`

File: deployments/types/common.json (hunk 1 above)
Comment (hbulgarini) [reply deployments/types/common.json:2]: ouh , i didn't notice that until now, fixing it

File: deployments/types/common.json [hunk 2]
```diff
@@ -134,5 +134,26 @@
 	"MessageFeeData": {
 		"lane_id": "LaneId",
 		"payload": "OutboundPayload"
-	}
+	},
+	"Precommit": {
+    		"target_hash": "BridgedBlockHash",
+    		"target_number": "BridgedBlockNumber"
+  	},
```
Comment (tomusdrw) [inline deployments/types/common.json:141]: The indentation is still a bit off.

**Review by tomusdrw — APPROVED** [review]
looks good, but the indentation is still a bit off.

**Review by tomusdrw — APPROVED** [review]

**Pull request merged**